archive/tar_windows.go

```diff
@@ -101,3 +141,310 @@ func setxattr(path, key, value string) error {
 	// since xattrs should not exist in windows diff archives
 	return errors.New("xattrs not supported on Windows")
 }
+
+func prepareApply() error {
+	return winio.EnableProcessPrivileges([]string{winio.SeBackupPrivilege, winio.SeRestorePrivilege})
+}
+
+func postApply(err error, isIsolated bool) error {
+	if !isIsolated {
+		if err := winio.DisableProcessPrivileges([]string{winio.SeBackupPrivilege, winio.SeRestorePrivilege}); err != nil {
```

Comment:
Do we want to remove this for now based on previous discussion? What would happen if 2 unpacks are happening simultaneously. Personally I would rather not have the archive package know about isolation and privileges and leave that to the higher level caller.

Reply:
Yes. I'll remove this for now.